<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="YearNotFoundExceptionTest" tests="8" skipped="0" failures="0" errors="0" timestamp="2022-08-04T14:50:25" hostname="fedora" time="0.043">
  <properties/>
  <testcase name="doesNotThrowAtGetName()" classname="YearNotFoundExceptionTest" time="0.001"/>
  <testcase name="doesNotThrowAtGetCount()" classname="YearNotFoundExceptionTest" time="0.002"/>
  <testcase name="doesNotThrowAtGetTotal()" classname="YearNotFoundExceptionTest" time="0.029"/>
  <testcase name="throwsAtGetRanking()" classname="YearNotFoundExceptionTest" time="0.002"/>
  <testcase name="throwsAtGetCount()" classname="YearNotFoundExceptionTest" time="0.001"/>
  <testcase name="throwsAtGetTotal()" classname="YearNotFoundExceptionTest" time="0.001"/>
  <testcase name="throwsAtGetName()" classname="YearNotFoundExceptionTest" time="0.001"/>
  <testcase name="doesNotThrowAtGetRanking()" classname="YearNotFoundExceptionTest" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
